src/app/pages/account/create/index.vue

```diff
@@ -58,6 +59,25 @@
         </template>
       </LayoutPage>
     </AppStep>
+    <AppStep v-slot="attributes" :is-active="index === -1">
+      <LayoutPage v-bind="attributes">
+        <LayoutPageResult type="error">
+          <template #description>
+            {{ errorDescription ?? t('tryAgain') }}
+          </template>
```

Comment:
I think we can move `errorDescription` to the default slot (which is not wrapped by `<template #default>` for brevity.
```suggestion
          {{ errorDescription }}
          <template #description>
            {{ t('tryAgain') }}
          </template>
```

Reply:
changed, you can resolve